i18n/zh-cn/docusaurus-plugin-content-docs/current/developer_guide/deployment.md

```diff
@@ -0,0 +1,141 @@
+---
+id: deployment
+title: 部署到 Kubernetes
+---
+
+<!--
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+
+部署YuniKorn的最简单方法是利用我们的[helm charts](https://hub.helm.sh/charts/yunikorn/yunikorn)，您可以在[这里](get_started/get_started.md)找到指南。本文档描述了手动部署YuniKorn调度器和准入控制器的过程，主要面向开发人员。
+
```

Comment:
There have new commit after this PR. ( 9a2f4a93d84b2e6c56a1c6e29a2aaaa1d66c3e48 )
Missing translation of 
> **Note** The primary source of deployment information is the Helm chart, which can be found at [yunikorn-release](https://github.com/apache/yunikorn-release/). Manual deployment may lead to out-of-sync configurations, see [deployments/scheduler](https://github.com/apache/yunikorn-k8shim/tree/master/deployments/scheduler)